config/schema/oe_corporate_site_info.schema.yml

```diff
@@ -11,3 +11,6 @@ oe_corporate_site_info.settings:
       sequence:
         label: 'SKOS entity ID'
         type: string
+    accessibility:
+      type: label
+      label: 'Accessibility statement'
```

Comment:
Put a description as well and a comment as to why we are making a link a type "label". We did this elsewhere as well.